tests/integration/targets/lookup_collection_version/runme.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: localhost
+  tasks:
+    - name: Test collection_version
+      assert:
+        that:
```

Comment:
It might be good to test `with_community.general.collection_version`, or testing the same thing by looking up multiple terms in a single lookup, a la:
```
lookup('community.general.collection_version', 'testns.testcoll', 'testns.testcoll_mf') == ['0.0.1', '0.0.1']
```
(but it would be better to have a test with different expected values)

Reply:
I added one test with multiple terms in c9102f50dc2891b2f177a410f517a052241ff4be (also includes a test for an invalid FQCN).